src/components/NotificationBar/index.tsx

```diff
@@ -81,22 +88,34 @@ export default function NotificationBar() {
         onClose={handleClose}
         MenuListProps={{ 'aria-labelledby': 'notification-menu-button' }}
       >
-        {notifications.filter((notification) => !notification.seen).length ? (
-          notifications
-            .filter((notification) => !notification.seen)
-            .map((notification) => (
+        {notifications.length ? (
+          notifications.map((notification) =>
+            notification.interacted ? (
               <MenuItem
                 key={notification.id}
                 onClick={() => {
-                  dispatch(appActions.seenNotification(notification));
+                  dispatch(appActions.interactedWithNotification(notification.id));
                   if (notification.source === 'node/message') {
                     navigate(`networking/messages${searchParams ? searchParams : ''}`);
                   }
                 }}
               >
                 {notification.name}
               </MenuItem>
-            ))
+            ) : (
+              <StyledUnreadMenuItem
+                key={notification.id}
+                onClick={() => {
+                  dispatch(appActions.interactedWithNotification(notification.id));
+                  if (notification.source === 'node/message') {
+                    navigate(`networking/messages${searchParams ? searchParams : ''}`);
+                  }
+                }}
+              >
+                {notification.name}
+              </StyledUnreadMenuItem>
+            ),
+          )
```

Comment:
Those 2 are the same. Just manipulate the styling with a className like: ```className={`notification.interacted ? 'unreadMenuItem' : ''`}``` and add in styled sth like:
```
const SMenuItem = styled(MenuItem)`
&.unreadMenuItem {
  background-color: rgba(0, 0, 180, 0.3);
  opacity: 90%;
  border-bottom: 1px solid #444;
  }
`;
```